plugin/interface_generator/main.go

```diff
@@ -310,6 +310,7 @@ func (s *apiRPCServer) {{.Name}}(args *{{.Name | obscure}}Args, returns *{{.Name
 		{{.Name}}{{funcStyle .Params}} {{funcStyle .Return}}
 	}); ok {
 		{{if .Return}}{{destruct "returns." .Return}} = {{end}}hook.{{.Name}}({{destruct "args." .Params}})
+		{{if .Return}}{{encodeErrors "returns." .Return}}{{end}}
```

Comment:
A little request for better readable code: Using the tool from https://golang.org/pkg/text/template/#hdr-Text_and_spaces It's possible to omit the empty line and reduce the `diff`.